frontend/src/Indexer/Add/SelectIndexerRow.js

```diff
@@ -26,7 +26,8 @@ class SelectIndexerRow extends Component {
     const {
       protocol,
       privacy,
-      name
+      name,
+      language
```

Comment:
You'll need to add `language` to propTypes in line 57 to appease lint. 

`language: PropTypes.string.isRequired,`

Reply:
Ah shoot, thanks, don't know how I missed that